### PR TITLE
Npm

### PR DESCRIPTION
**Pull request by dmadisetti**
Cool, is there anyway the team can publish? Or is it tied to you? What's the workflow for publishing? https://docs.npmjs.com/cli/owner

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "gameframe",
+  "version": "1.0.4",
+  "description": "Wrapper for PhysicsJS, which make basic game development as simple as possible.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "CarolinaIgnites",
+  "license": "ISC",
```
Comment (dmadisetti) [inline package.json:10]: Repo has MIT license listed

File: package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "gameframe",
+  "version": "1.0.4",
+  "description": "Wrapper for PhysicsJS, which make basic game development as simple as possible.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (dmadisetti) [inline package.json:7]: It'd be cool if we could set up basic unit testing. It looks like mocha is pretty good: https://mochajs.org/

Ideally, I'd say make sure we can import everything as very first unit test

File: index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = Object.assign(
+	{},
+	require('./src/js/gameframe'),
+	require('./src/js/interactive-custom')
```
Comment (dmadisetti) [inline index.js:4]: Shouldn't we require physicsjs as well? Or should gameframe import it?

**Comment by kmsalah** [comment]
Ready to go, works with PR https://github.com/CarolinaIgnites/editFrame/pull/31

Added some basic testing with mocha. Didn't go too far with the tests but set it up and tested loading gameframe.js

**Pull request merged**